src/main/java/me/senseiwells/essentialclient/clientscript/values/WorldValue.java

```diff
@@ -1,639 +1,675 @@
-package me.senseiwells.essentialclient.clientscript.values;
```

Comment:
What happened here? Was this meant to be completely different, I can't see any changes so I'm not sure what happened here.

Reply:
Intellij error, or some weird thing.... not sure what happened